packages/platform/platform-http/src/testing/FakeResponse.ts

```
@@ -25,6 +39,35 @@ export class FakeResponse extends EventEmitter {
+    if (opts.path == null) {
+      opts.path = "/";
+    }
+
+    this.append("Set-Cookie", cookie.serialize(name, String(val), opts));
+
+    return this;
+  }
+
+  clearCookie(name: string, options?: TsED.SetCookieOpts) {
+    const opts = {path: "/", ...options, expires: new Date(1)};
+    delete opts.maxAge;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)